master/internal/rm/kubernetesrm/kubernetes_resource_manager.go

```diff
@@ -246,42 +223,161 @@ func (k *kubernetesResourceManager) Receive(ctx *actor.Context) error {
 			k.config.MasterPort,
 		)
 
-		k.pools[KubernetesDummyResourcePool] = ctx.MustActorOf(
-			KubernetesDummyResourcePool, newResourcePool(k.config, k.podsActor),
-		)
+		for _, poolConfig := range k.poolsConfig {
+			poolConfig := poolConfig
+			k.pools[poolConfig.PoolName] = ctx.MustActorOf(
+				poolConfig.PoolName, newResourcePool(k.config, &poolConfig, k.podsActor),
+			)
+		}
 
-	case
-		tasklist.GroupActorStopped,
-		sproto.SetGroupMaxSlots,
-		sproto.SetAllocationName,
-		sproto.AllocateRequest,
-		sproto.ResourcesReleased,
-		sproto.UpdatePodStatus,
-		sproto.PendingPreemption,
-		sproto.GetJobQ,
```

Comment:
somewhat related think CLI has a bug of not passing ```args.resource_pool``` in ```det job ls``` 

https://github.com/determined-ai/determined/blob/b69bab1c1c59172963ec0f8eb2caba6cc6c24c1a/harness/determined/cli/job.py#L31

```
                    Arg(
                        "-p",
                        "--resource-pool",
                        type=str,
                        help="The target resource pool, if any.",
                    ),

```